File: TestCase.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC
from webdriver_manager.chrome import ChromeDriverManager

driver = webdriver.Chrome(ChromeDriverManager().install())
driver.get("https://myt-sit.fhcdn.dev")

wait = WebDriverWait(driver, 10)

login = wait.until(EC.presence_of_element_located((By.ID, "login")))
login.send_keys("7033443434")

continue_button = wait.until(EC.element_to_be_clickable((By.ID, "continue")))
continue_button.click()

File: TestCase.java


public class Test1 {

WebDriver driver;

@BeforeMethod

public void setUp() {

System.setProperty("webdriver.chrome.driver", "C:\\chromedriver.exe");

driver = new ChromeDriver();

driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);

}

@Test

public void test1() {

driver.get("https://myt-sit.fhcdn.dev/auth/login");

driver.findElement(By.id("username")).sendKeys("703344343");

driver.findElement(By.id("continue")).click();

}

@AfterMethod

public void tearDown() {

driver.quit();

}

}